package.json

```diff
@@ -22,13 +22,17 @@
     {
       "name": "Shravan Sunder",
       "url": "https://github.com/ShravanSunder"
+    },
+    {
+      "name": "Felipe Novaes Rocha",
+      "url": "https://github.com/kamikazebr"
     }
   ],
   "scripts": {
     "watch": "ts-patch i && tsc -w --noEmit",
     "clean": "shx rm -rf ./lib",
     "build:old": "yarn clean && ts-patch i && yarn shx mkdir lib && yarn shx cp -r src lib/src.ts && tsc -p tsconfig.build.json",
-    "es-build": "yarn clean && node esbuild.config.js && tsc -p tsconfig.build.json",
+    "es-build": "yarn clean && node esbuild.config.js && tsc -p tsconfig.build.json && shx cp package.json ./lib/package.json",
```

Comment:
I added to make my yarn link work, but i can remove it for sure and use the yalc you told me